external-declarations/src/code-mod/code-transform.ts

```diff
@@ -54,7 +54,7 @@ function findNearestParentWithTypeAnnotation(node: ts.Node): ts.Node | undefined
 export function addTypeAnnotationTransformer(sourceFile: ts.SourceFile, program: ts.Program, moduleResolutionHost?: ts.ModuleResolutionHost) {
     const typeChecker = program.getTypeChecker();
     const nodesToFix = new Map(program.getDeclarationDiagnostics(sourceFile).
-                                filter((diag) => diag.code === 9007).
+                                filter((diag) => diag.code === 9007 || diag.code === 9009).

```

Comment:
I'm guessing I should write a fix for the new error code `9009` by adding declarations to property assignments happening in functions?